src/components/subscriptions/data/hooks.js

```diff
@@ -29,9 +30,34 @@ export const useSubscriptions = ({ enterpriseId, errors, setErrors }) => {
   const [loading, setLoading] = useState(true);
 
   useEffect((page = 1) => {
-    LicenseManagerApiService.fetchSubscriptions({ enterprise_customer_uuid: enterpriseId, page })
+    LicenseManagerApiService.fetchCustomerAgreementData({ enterprise_customer_uuid: enterpriseId, page })
       .then((response) => {
-        const { data: subscriptionsData } = camelCaseObject(response);
+        const { data: customerAgreementData } = camelCaseObject(response);
+
+        const subscriptionsData = {
+          results: [],
+          count: 0,
+          next: null,
+          previous: null,
```

Comment:
nit: can this initial state be shared between this code and the `useState` line above maybe via some constant like INITIAL_SUBS_DATA or so?